spec/spec_helper.rb

```diff
@@ -0,0 +1,6 @@
+require 'cog'
+require 'rspec/cog'
```

Comment:
I think that's the exact one I cribbed from. I don't particularly care much either way, though.